src/main.rs

```diff
@@ -21,6 +20,10 @@ use tracing_subscriber::FmtSubscriber;
 #[derive(StructOpt, Clone, Debug)]
 #[structopt(name = "comtrya")]
 pub struct Opt {
+    /// Performs a dry-run without changing the system
+    #[structopt(short = "n", long)]
```

Comment:
Let’s remove the short option, I don’t think that flag is intuitive and I don’t wanna add something I can’t remove later 😀

Reply:
Done